frontend/src/app/pages/exercises/exercise/shared/editor-panel/create-vehicle-template-modal/open-create-vehicle-template-modal.ts

```diff
@@ -0,0 +1,8 @@
+import type { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { CreateVehicleTemplateModalComponent } from './create-vehicle-template-modal.component';
+
+export async function openCreateVehicleTemplateModal(
+    ngbModalService: NgbModal
+) {
+    ngbModalService.open(CreateVehicleTemplateModalComponent);
```

Comment:
Would you mind using an a bit larger size (maybe `lg`) for both the create and edit modal?

When editing a more complex template (e.g. GW San) with lots of material and personnel, one has to scroll down quite a bit to reach the buttons, as only two items fit next to each other.